extra/sodium/sodium-docs.factor

```diff
@@ -0,0 +1,15 @@
+! Copyright (C) 2020 Alexander Ilin.
+! See http://factorcode.org/license.txt for BSD license.
+USING: help.markup help.syntax kernel sodium.ffi ;
+IN: sodium
+
+HELP: check-malloc
+{ $values
+  { "ptr" null }
```

Comment:
I don't think you mean ``null`` here.  Since the code is checking for truthiness.

Reply:
You're right, that was autogenerated stub.